ES/ESdd032.xml

```diff
@@ -148,207 +152,213 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                           l.6, 48vc l.8, 53rc l.3, 57rc l.19, 60rc l.7, 60va l.11, 60va l.18, 61ra
-                           l.6, 61rc l.10, 65ra l.17, 65rc l.6, 65rc l.17, 65vb l.17, 65vc l.14,
-                           66vc l.6, 67vc l.19, 71rc l.5, 71rc l.10, 71rc l.16, 72ra l.16, 73rc l.5,
-                           73vb l.5, 73vc l.10, 75vb l.11, 75vc l.4, 76rb l.4, 78vb l.11, 79vc l.13,
-                           85rc l.18, 86ra l.11, 86va l.1, 88ra l.3, 89ra l.2, 90rc l.9, 92ra l.1,
-                           105ra l.10, 105vb l.2, 109ra l.2, 111ra l.5, 111ra l.9, 112ra l.1, 118rb
-                           l.8, 121vc l.10, 124rb l.4, 126ra l.12, 126rc l.2, 126rb l.8, 133rb l.12,
-                           133vc l.1, 139rb l.6, 160rb l.16, 169rb l.12, 169rb l.13.&#xD; -
-                           Tamallas-signs, in the shape of two perpendicular lines, indicating the
-                           place where the omissions, written in the margin in a secondary hand,
-                           belong: <locus target="#27rc">27rc</locus> l.2, 62rc l.19, 62vc l.19,
-                           68va l. 2, 68vc l.8, 69rc l.3, 72ra l.11,100rb l.16, 107ra l.12, 113va
-                           l.8, 131vb l.5, 131vc l.6, &#xD; - Signs circled with dots, in the shape
-                           of a cross (occasionally with more than 4 arms) with a loop at the end of
-                           each arm, are written in the intercolumns and side margins of Text I:
-                           e.g., <locus target="#2v">2v</locus>, 4r, 5r, 6v, 7r, 7v, 8v, 9r, 13v,
-                           14v, 16r, 21v, 29v, 30v, 41r, 47r, 47v, 48r, 52r. &#xD; - The name of the
-                           church of ʾAbuna ʾAragāwi, za-ʾabuna ʾAragāwi, is written under Additio
-                           1. &#xD; - A short note (5 lines) has been erased from <locus target="#172va">172va</locus>.&#xD; - Pen trials: <locus target="#1r">1r</locus>, 36r, 173v.&#xD; - Minor notes, writing exercises: <locus target="#4r">4r</locus>, 11rv, 12rv, 13v, 14r, 15rv, 19v, 23r, 55v,
-                           56r, 172vb.</item>
-                        <item xml:id="e2"><!--2019-04-02+02:00 Updated DD-032 
-            from DomLib -->
-                           <desc type="findingAid">- Dates of readings are written in red in the lower
-                              margin of some fols.: ʾamä 30-hu lä-mäskäräm zǝmmare: <locus target="#69vc">fol. 69vc</locus> ʾamä 4 (or 9) lä-Tǝqǝmt: <locus target="#70rb">fol. 70rb</locus>.</desc>
+                        <item xml:id="e12">
+                           <desc type="findingAid">Dates of readings are written in red in the lower
+                              margin of some folia.: <foreign xml:lang="gez">ʾama 30-hu la-maskaram zǝmmāre</foreign> on <locus target="#69vc"/>, 
+                              <foreign xml:lang="gez"> ʾama 4 (or 9) la-Tǝqǝmt</foreign> on <locus target="#70rb"/>.</desc>
```

Comment:
Ṭǝqǝmt (and Maskaram is spelled small here, I would opt for capitals in both cases)